src/lib/openapi/spec/group-schema.ts

```diff
@@ -7,49 +7,67 @@ export const groupSchema = {
     type: 'object',
     additionalProperties: true,
```

Comment:
This is a response schema, right? In that case, this should be `false`
```suggestion
    additionalProperties: false,
```

Reply:
This one is shared for request and response schema. Once I get to the create schemas I may split those two. But for now I only wanted to add examples/descriptions